colony.js

```diff
@@ -0,0 +1,13 @@
+function Colony() {
```

Comment:
It feels like the colony should be doing at least all the ant setup (and possible the nest setup as well) rather than the sketch file. I feel like the sketch should simply instantiate the colony and call a function on the colony which will set itself up